src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -96,6 +102,9 @@ const styles = StyleSheet.create({
     position: 'absolute',
     bottom: 6,
     right: 8
+  },
+  ml8: {
+    marginLeft: 8
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new style `ml8` is introduced. Make sure it's being used in the component, otherwise, it's an unnecessary addition.



<!-- This is an auto-generated comment by OSS CodeRabbit -->